.github/workflows/ci.yml

```diff
@@ -35,15 +67,15 @@ jobs:
       id-token: write # needed for the Vault authentication
 
     steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
       - name: Load Secrets from Vault
```

Comment:
this step is failing https://github.com/rancher/cli/actions/runs/8879832219/job/24378515349?pr=362

Reply:
Yes, that's expected because the action is running on a fork (`Unable to get ACTIONS_ID_TOKEN_REQUEST_URL env variable`).

The merged version is fine: https://github.com/rancher/cli/pull/365#issuecomment-2075210066

I could add a check to see if the action is running on the same repo:

```yaml
if: github.repository_owner == 'rancher'
```

Reply:
wouldn't it be better to just execute it when pushing to a release branch or when creating a tag? We could create a separate workflow, something like we do in kubewarden https://github.com/kubewarden/kubewarden-controller/blob/main/.github/workflows/fossa.yml

Reply:
wouldn't it be better to execute it just when pushing to a release branch or when creating a tag? We could create a separate workflow for fossa like we do in kubewarden https://github.com/kubewarden/kubewarden-controller/blob/main/.github/workflows/fossa.yml

Reply:
Yes, I like it more!

Reply:
Done 3672f7c.